packages/core/src/config/__tests__/__snapshots__/config-resolvers.test.ts.snap

```diff
@@ -63,7 +63,7 @@ exports[`resolveConfig > should ignore minimal from the root and read local file
     "no-required-schema-properties-undefined": "off",
     "no-schema-type-mismatch": "warn",
     "no-unresolved-refs": "error",
-    "operation-2xx-response": "warn",
+    "operation-2xx-response": "off",
```

Comment:
The actual resolution order has changed compared to v1. I think it should be somehow reflected in the changeset.

Reply:
I'll update changelog